contracts/Vault.sol

```diff
@@ -58,6 +61,10 @@ contract Vault is ERC4626Metadata, Ownable {
         return 0;
     }
 
+    function getRate() external view returns (uint256) {
```

Comment:
we could just have `_rate` be public and save a getter

Reply:
@brettvniekerk If it's only used externally, defining an external function is cheaper than making it public

Reply:
Also needs to be on the Interface for external contracts.